doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -48,7 +48,11 @@ This specification is licensed under the
 
 ## Conformance
 
-All the interfaces described by this specification are mandatory and must be implemented in order to be compliant with. 
+This specification describes the "full" compliance level.
+
+However, next iterations of this specification could define optionality
+in some parts and the compliance level (or roles/profiles)
+corresponding to them.
```

Comment:
do we need this here? I'm not convinced of optionality that is usually a big interoperability hole. I would suggest this statement is added to release notes but not here